index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css" integrity="sha512-YWzhKL2whUzgiheMoBFwW8CKV4qpHQAEuvilg9FAn5VJUDwKZZxkJNuGM4XkWuk94WCrrwslk8yWNGmY1EduTA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <link rel="stylesheet" href="app.css">
-    <title>Freddy KABERUKA</title>
+    <title>Freddy KABERUKA Portofolio</title>
```

Comment:
### Kindly add a report in the PR description that includes the following:
- The report includes a check about page titles.
- The report includes a check about image text alternatives (`<img alt="alt text">`).
- The report includes a check about text headings.
- The report includes a check about ** color contrast**.
- The report includes a check about resizing.
- The report includes a check about interaction.
- The report includes a check about moving content.
- The report includes a check about multimedia.
- The report includes a check on the basic structure of the page.

### The result section of the report should include a statement pertained to every point checked:
- If everything was ok, Please make a statement about that (e.g. "Page titles - no issues found").
- If issues were found, Please link to the corresponding line of code in the Pull Request changes.